glide.yaml

```diff
@@ -6,7 +6,6 @@ owners:
   email: rob@tigera.io
 import:
 - package: github.com/docopt/docopt-go
-- package: github.com/ghodss/yaml
```

Comment:
I think given that there is an interface change and you can't switch between the two, I don't  think that would be a preferred approach.